File: 프로그래머스/1/12921. 소수 찾기/소수 찾기.py
def solution(n):
    answer = 0
    
    nums = [False for _ in range(n + 1)]
    
    for i in range(2, n + 1):
        if not nums[i]:
            answer += 1
            
            for j in range(2, n // i + 1):
                nums[i * j] = True
    
    return answer

File: 프로그래머스/1/12921. 소수 찾기/소수 찾기.js
function solution(n) {
    const primeNumbers = Array(n + 1).fill(false);
    
    let answer = 0;
    for (let i = 2; i <= n; i++) {
        if (!primeNumbers[i]) {
            answer++;
            let multipleNumber = 2;
            while (i * multipleNumber <= n) {
                primeNumbers[i * multipleNumber] = true;
                multipleNumber++;
            }
        }
    }
    
    
    return answer;
}